app/bundles/EmailBundle/Form/Type/ConfigType.php

```diff
@@ -608,6 +608,78 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ]
         );
 
+        $builder->add(
+            'mailer_rabbitmq_use_ssl',
+            YesNoButtonGroupType::class,
+            [
+                'label'      => 'mautic.email.config.mailer.rabbitmq_use_ssl',
+                'label_attr' => ['class' => 'control-label'],
+                'attr'       => [
+                    'class'        => 'form-control',
+                    'data-show-on' => $rabbitMQSpoolShowConditions,
+                    'data-hide-on' => $rabbitMQSpoolHideConditions,
+                    'tooltip'      => 'mautic.email.config.mailer.rabbitmq_use_ssl.tooltip',
+                    'onchange'     => 'Mautic.disableSendTestEmailButton()',
+                ],
+                'data'       => empty($options['data']['mailer_rabbitmq_use_ssl']) ? false : true,
+                'required'   => false,
+            ]
+        );
```

Comment:
feels like his could be a method. you're only changing minimal things in the parameters to add.

Reply:
This is just using the Mautic standard form factory, used across all the form pages on the app. Refactoring that is very low on my list of todos.